src/reducers/secrets.js

```diff
@@ -14,14 +14,28 @@ limitations under the License.
 import { combineReducers } from 'redux';
 import merge from 'lodash.merge';
 import { ALL_NAMESPACES } from '@tektoncd/dashboard-utils';
-import { isStale } from '../utils';
+
+// TODO: replace isSecretStale() with utils.js isStale() when secrets store uses
+// byId structure.
```

Comment:
> when secrets store uses byId structure

what's stopping us from doing this?

Reply:
Nothing, but I wouldn't know how to do it and I thought it was better to have a temporary solution in place to avoid the errors. I was going to make a follow up issue after this one was merged.

Reply:
no problem, can you link the issue here when you create it?